doc/conf.py

```diff
@@ -416,17 +416,16 @@
 latex_elements = {
     "classoptions": ",oneside",
     "babel": "\\usepackage[english]{babel}",
+    "preamble": r"""
+\usepackage{amsmath}\usepackage{amsfonts}\usepackage{bm}\usepackage{morefloats}
+\let\oldfootnote\footnote
+\def\footnote#1{\oldfootnote{\small #1}}
+""",
```

Comment:
Am actually trying to figure out  how much of our latex config is required given we do not generate pdf version of our doc.

So this may actually end up being deleted.